MapboxNavigation.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,62 +6,62 @@
         "repositoryURL": "https://github.com/mapbox/mapbox-common-ios.git",
         "state": {
           "branch": null,
-          "revision": "49cb6783a3ae9e1a4bcda66e48e39d2c5c8cc4c5",
-          "version": "10.0.0-beta.9.2"
+          "revision": "a1ffbf8abecef7637b458dd2e561500854757705",
+          "version": "10.0.0-beta.12"
```

Comment:
This changes should also take place in the Package.resolved in the root directory. Xcode will do that automatically if you open the Package.swift as its own workspace.

Reply:
When I do this (and clear package caches, of course), I get the following error:

> 
checksum of downloaded artifact of binary target 'MapboxNavigationNative' (feefa910c95e1dead8668a7d8a871b243a3fa688fed85b66c1483d0c33254bb1) does not match checksum specified by the manifest (8fd54eee4277f1327015cc0bcaed8a878bf44d1804364cd5d93dfab9e2d1a5af)

Reply:
https://github.com/mapbox/mapbox-navigation-native-ios/commit/3d6221c357cd9f7604dd384bdd9cb1286fe44ea9#r47903151